app/models/search_request.rb

```diff
@@ -13,6 +13,7 @@ def self.fromRequest(rparams)
     obj.seq = rparams[:seq]
     obj.showCount = rparams[:showCount]
     obj.pageNum = rparams[:pageNum]
+    obj.tagName = rparams[:tagName]
```

Comment:
here, we can add some conditions like if rparams[:tagName].blank? etc for checking if tagName is present or not. 
But, I think it is ok to set tagName as nil if not given, as in some responses we need to checking tagName for nil if not present to pass the test successfully. Also, it will keep the json response fields uniform.

Reply:
👍